liblll/CMakeLists.txt

```diff
@@ -11,6 +11,7 @@ set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -DSTATICLIB")
 
 aux_source_directory(. SRC_LIST)
 
+include_directories(BEFORE ${JSONCPP_INCLUDE_DIRS})
```

Comment:
llblll is using libdevmcore, I have includes there but dependences seems to not work correctly.